docs/UserGuide.md

```diff
@@ -102,8 +102,15 @@ Format: `add r/ROLE n/NAME [p/PHONE] e/EMAIL [d/DESCRIPTION]`
 * `PHONE` is the phone number of the contact.
 * `EMAIL` is the email address of the contact.
 * `DESCRIPTION` contains additional information about the contact, such as their job position.
-* You cannot add contacts with the same name and at least one other identity field (phone or email) that is the same,
-e.g. `add r/dev n/John p/123` will not work if a contact with the name `John` and phone `123` exists in the contact list.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** You cannot add a contact with the same name and email as another contact in your 
```

Comment:
Suggest to edit to Chrystal's phrasing: `You cannot add a contact with the same name and email as another existing contact`. Here and elsewhere for the other commands/models.

Reply:
May want to leave out the example to keep the `Note` short. What do you think?